contrib/cirrus/lib.sh.t

```diff
@@ -0,0 +1,149 @@
+#!/bin/bash
```

Comment:
Is tis file named correctly?

Reply:
Yes: `.t` for `test`. I wrote it to see what the existing code was actually doing, and am committing it so that we don't break it. PS thanks for the reminder, in all today's excitement I forgot about this.

Reply:
I would very much like to see this test actually run for PRs.  We do a bunch of other "CI sanity" checks already in `contrib/cirrus/prebuild.sh` (only on the latest Fedora VM). Would you consider adding a run of this, there?